docs/topics/logging.txt

```diff
@@ -261,12 +265,37 @@ First, here's a simple configuration which writes all request logging from the
 If you use this example, be sure to change the ``'filename'`` path to a
 location that's writable by the user that's running the Django application.
 
-Second, here's an example of a fairly complex logging setup, configured using
+Second, here's an example of how to make the logging system print all of
+Django's logging to the console based on the value of an environment variable.
+It overrides the fact that ``django.request`` and ``django.security`` don't
+propagate their log entries. It may be useful during local development. Set the
+environment variable ``DJANGO_LOG_LEVEL=DEBUG`` to see all of Django's logging
+calls in the console::
+
+    import os
+
+    LOGGING = {
+        'version': 1,
+        'disable_existing_loggers': False,
+        'handlers': {
+            'console': {
+                'class': 'logging.StreamHandler',
+            },
+        },
+        'loggers': {
+            'django': {
+                'handlers': ['console'],
+                'level': os.getenv('DJANGO_LOG_LEVEL', 'INFO'),
+            },
+        },
+    }
+
+Finally, here's an example of a fairly complex logging setup, configured using
```

Comment:
removed "configured using dictConfig"

It's said above, "By default, Django uses the `dictConfig format`_." and below, "The LOGGING_CONFIG setting defines the callable that will be used to configure Django’s loggers. By default, it points at Python’s logging.config.dictConfig() function"